src/skottie-player.ts

```diff
@@ -588,9 +631,14 @@ export class SkottiePlayer extends LitElement {
     const files = assetInput.files;
     const file = files?.[0];
     if (file && mapKey) {
+      if (mapKey === 'img_3.png') {
+        return;
+      }
       const reader = new FileReader();
       if (mapKey === 'Image_0.jpeg') {
         this.updateColors(reader, file, mapKey);
+      } else if (mapKey === 'img_1.png') {
```

Comment:
'img_1.png' -> Consider making this string an input variable. 

We could call the image that's segmented "source_image", and the new image created from the segmentation process "mask", etc.